Fix TypeError for `asdict(<class with namedtuple>, retain_collection_types=True)` (#1165)
python-attrs/attrs: changelog.d/1165.change.md

> Fixed `TypeError` when `asdict`/`astuple` is called for classes that have a `namedtuple` field with parameter `retain_collection_types=True`.

Reviewer: ```suggestion
Fixed serialization of namedtuple fields using `attrs.asdict/astuple()` with `retain_collection_types=True`.
```
